### DISCUSSION TITLE
help-how to prepare data from CP2K molecular dynamics dataset

### DISCUSSION THREAD

**Discussion by chandagsin** [deepmodeling/dpdata#412] [Q&A]
Dear Developers, 

I want to convert my CP2K MD dataset to a DeePMD readable format. I have .out file that records energies, cell sizes etc. but not the coordinates of each frame, and .xyz file that records the coordinates and energies. I have tried 
data = dpdata.LabeledSystem('xxx.out', fmt = 'cp2k/output') 
data = dpdata.LabeledSystem('xxx.out', fmt = 'cp2k/aimd_output') 
data = dpdata.LabeledSystem('xxx.xyz', fmt = 'cp2k/quip/gap/xyz') 

none of them works...What command should I type in?
Thanks, 
chandagsin

**Comment by PolyuWeldingSpock** [comment]
Hi,
what cp2k version do you use?

**Comment by robinzyb** [comment]
Use cp2kdata plugin instead. If this disscussion is not active any more I will close it tomorrow
